test/integration/test_http.py

```diff
@@ -229,11 +228,11 @@ def test4():
 
 
 def test5():
-    """Basic GET without Host on 0.9 - Expect 505 version not supported"""
+    """Basic GET without Host on 0.9- Expect 404"""
     x = ruleset.Input(dest_addr='example.com', version='HTTP/0.9', headers={})
     http_ua = http.HttpUA()
     http_ua.send_request(x)
-    assert http_ua.response_object.status == 505
+    assert http_ua.response_object.status == 404
```

Comment:
Perhaps we can remove this test altogether. It doesn't look like it's providing much value tbh.

Reply:
Test removed